Makefile

```diff
@@ -19,3 +23,20 @@ markdown-toc:
 			echo markdown-toc: no TOC markers, skipping $$f; \
 		fi; \
 	done
+
+.PHONY: markdown-link-check
+markdown-link-check:
+	@if ! npm ls markdown-link-check; then npm install; fi
+	@for f in $(ALL_DOCS); do \
+		npx --no -- markdown-link-check --quiet --config .markdown_link_check_config.json $$f \
+			|| exit 1; \
+	done
+
+.PHONY: markdownlint
+markdownlint:
+	@if ! npm ls markdownlint; then npm install; fi
+	@for f in $(ALL_DOCS); do \
+		echo $$f; \
+		npx --no -p markdownlint-cli markdownlint -c .markdownlint.yaml $$f \
```

Comment:
```suggestion
		npx --no -- -p markdownlint-cli markdownlint -c .markdownlint.yaml $$f \
```

Reply:
I am on npm/npx 8.3.1 and with this change it is failing for me:
```
./CHANGELOG.md
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/-p - Not found
npm ERR! 404 
npm ERR! 404  '-p@latest' is not in this registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

Reply:
I'd prefer to keep this code in sync with https://github.com/open-telemetry/opentelemetry-specification (unless you know how to make it work for either version of npm).

Reply:
Upgraded to `node 16.15.0` and it worked fine.